CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Change history for ui-data-export
 
 ## 2.1.0 (IN PROGRESS)
+* Refactor to miragejs from bigtest/mirage. UIDEXP-137.
```

Comment:
This one should be at the bottom right after `Add job profiles deleting. UIDEXP-111.`